src/standalone/App.js

```diff
@@ -0,0 +1,38 @@
+import { NotFound } from "../lib/components/ui";
```

Comment:
I see there is an App here, then you should remove the App from the lib, it should not have an app.
IMHO, I would prefer `demo` or similar to `standalone`, not sure I understand what that means.